src/js/covidMap.js

```diff
@@ -585,6 +558,37 @@ fluid.defaults("fluid.covidMap.map", {
     }
 });
 
+// When the location button on the mobile view is activated.
+fluid.covidMap.map.locationButtonOnMobileActivate = function (map, citiesList) {
+    if (citiesList.expandButton.model.expanded) {
```

Comment:
Please could you factor this into "toggle interaction + remainder" as with the listener below